google/cloud/README.md

```diff
@@ -20,7 +20,19 @@ Documentation for the common utilities is available [online][doxygen-link].
 
 ## Release Notes
 
-### v0.11.x - TBD
+### v0.12.x - TBD
+
+### v0.11.x - 2019-09
+
+* feat: Use macros for compiler id and version (#2937)
+* fix: Fix bazel build on windows. (#2940)
+* chore: disabled google-runtime-int (#2958)
```

Comment:
I would remove that, it was just a .clang-tidy config

Reply:
Done.